Source/cmake/OptionsJSCOnly.cmake

```diff
@@ -94,7 +100,10 @@ if (WIN32)
 endif ()
 
 string(TOLOWER ${EVENT_LOOP_TYPE} LOWERCASE_EVENT_LOOP_TYPE)
-if (LOWERCASE_EVENT_LOOP_TYPE STREQUAL "glib")
+if (WIN32)
```

Comment:
Would it make more sense for this to replace the else rather than be first?

Reply:
If I understand it correctly, `glib` and the else case corresponds to *nix platforms. So having the `WIN32` condition first earlier made it easier order.